### ISSUE TITLE
[V2] [theme] color options have no effect

### ISSUE THREAD

**Issue by zoidby** [woefe/ytcc#57]
Changing any of these doesn’t have an effect with V2:

```
[theme]
prompt_download_audio = 2
prompt_download_video = 4
prompt_play_audio = 2
prompt_play_video = 4
prompt_mark_watched = 1
table_alternate_background = 245
```

**Comment by woefe** [comment]
Thanks for your report! Should be fixed in 5e65787. Can you test it and let me know if it's working now?

**Comment by zoidby** [comment]
Yes, that works.

Thank you!
